ner/src/main/lbj/LbjTagger.lbj

```diff
@@ -54,8 +61,12 @@ real% WordEmbeddingFeatures(NEWord word) <-
   		for (; w != last; w = (NEWord) w.next) {
   			double[] embedding=WordEmbeddings.getEmbedding(w);
 			if(embedding!=null)
-				for(int dim=0;dim<embedding.length;dim++)
-					sense "place"+i+"dim"+dim : embedding[dim];
+				for(int dim=0;dim<embedding.length;dim++){
+				    sense "place"+i+"dim"+dim : embedding[dim];
+				    if(ParametersForLbjCode.currentParameters.useFE) {
+    				    sense word.domainName+"place"+i+"dim"+dim : embedding[dim];
+    				}
+				}
```

Comment:
I have checked the code in local. All the format is correct. I don't know why it becomes this here. I will fix them manually.